src/main/java/ch/njol/skript/aliases/ItemType.java

```diff
@@ -391,7 +391,26 @@ public void sendBlockChange(Player player, Location location) {
 			BlockUtils.sendBlockChange(player, location, blockType, d.getBlockValues());
 		}
 	}
-	
+
+	/**
+	 * Send a multi-block change to a player.
+	 * <p>This will send a packet for each chunk modified to the player, and will not change the block on the server.</p>
+	 * Requires MC 1.19.
+	 *
+	 * @param player Player to send change to
+	 * @param locations Array of locations of blocks to change
+	 * @param suppressLightUpdates Whether to suppress light updates.
+	 */
+	public void sendBlockChanges(Player player, Location[] locations, boolean suppressLightUpdates) {
```

Comment:
Let's take advantage of varargs.
```suggestion
	public void sendBlockChanges(Player player, boolean suppressLightUpdates, Location... locations) {
```